frontend/src/app/data-entry/object/object.component.html

```diff
@@ -0,0 +1,179 @@
+        <li class="breadcrumb-item active">
+            Letter from Radegund
+        </li>
+    </ol>
+</nav>
+
+<h1 class="mb-4">
+    Letter from Radegund
+    <small class="text-body-secondary">(<i>Fortunatus, Life of Radegund</i>)</small>
+</h1>
+
+<p class="lead">
+    A letter from Radegund describing her vision
+</p>
+
+<div ngbAccordion [closeOthers]="true">
+    <div ngbAccordionItem>
+        <h2 ngbAccordionHeader>
+            <button ngbAccordionButton>Identification</button>
+        </h2>
+        <div ngbAccordionCollapse>
+            <div ngbAccordionBody>
+                <ng-template>
+                    <div class="mb-3">
+                        <label class="form-label">
+                            Name
+                        </label>
+                        <input type="text" class="form-control" value="Letter from Radegund">
+
+                        <p>
+                            <small class="text-body-secondary">
+                                This is a name to identify the object in the overview of the source.
+                                For example: "letter from Radegund to Germanus", "Radegund's first letter", "a ring".
+                            </small>
+                        </p>
+                    </div>
+                    <div class="mb-3">
+                        <label class="form-label">
+                            Description
+                        </label>
+                        <input type="text" class="form-control" value="A letter from Radegund describing her vision">
+
+                        <p>
+                            <small class="text-body-secondary">
+                                Add a description to help identify this object
+                            </small>
+                        </p>
+                    </div>
+                    <div class="mb-3">
+                        <label class="form-label">
+                            Is this object a letter or a gift?
+                        </label>
+                        <select class="form-select">
+                            <option selected>Letter</option>
+                            <option>Gift</option>
+                        </select>
+                    </div>
+                </ng-template>
+            </div>
+        </div>
+    </div>
+    <div ngbAccordionItem>
+        <h2 ngbAccordionHeader>
+            <button ngbAccordionButton>Description in source text</button>
+        </h2>
+        <div ngbAccordionCollapse>
+            <div ngbAccordionBody>
+                <ng-template>
+                    <div class="mb-5">
+                        <h3>Sender</h3>
+                        <div class="mb-4">
+                            <label class="form-label">Who is described as the (original) sender?</label>
+                            <div class="form-check">
+                                <input type="checkbox" class="form-check-input" checked>
```

Comment:
Instead of checkboxes, this should probably become a dropdown menu in the eventual version.

(Out of scope for now, but good to think about: will we allow inline creation of agents? - If not, the user should be able to save a draft version of this object and come back to it later.)